Code/GraphMol/SubstructLibrary/SubstructLibrary.cpp

```diff
@@ -304,6 +344,15 @@ unsigned int SubstructLibrary::countMatches(const ROMol &query,
                       useQueryQueryMatches, numThreads);
 }
 
+unsigned int SubstructLibrary::countMatches(const TautomerQuery &query,
```

Comment:
See my comment in the header about making this (and the overload above it) a simple template which is defined in the header.